Adding dockerfile

abhay-agarwal opened the pull request:
Change your *-site.xml files before building your docker file (for now)

elingg committed:
Updated Formatting

This uses the current Mesosphere Java style.

elingg committed:
Update formatter settings

Allow for more readable wrap,
Comments are now 100 lines,
Add license,
Remove java compiler settings from formatter

elingg committed:
Fix comment

elingg committed:
Update formatting after review

elingg committed:
alphabetize formatter

elingg committed:
Merge pull request #18 from mesosphere/format-branch

Updated Formatting

elingg committed:
Deleting classes that are currently not in use

Simplifying the code by deleting the classes that aren’t in use as part
of the refactor.  Backup code can be added into the refactored design
once ready.  Currently, this code is not in use and is not functional.

adam-mesos committed:
Merge pull request #19 from mesosphere/refactor-classes

Deleting classes that are currently not in use

elingg committed:
Add multiple executors

Refactoring / clean up
Adding executors instead of framework messages

elingg committed:
update code per review

elingg committed:
fix order of imports

elingg committed:
Test correct functionality in config

elingg committed:
Consistent ordering for shell commands

elingg committed:
Disable hostname check

DNS may not be configured in which case we want to use IP not hostname

elingg committed:
Update Scheduler / Executor

- Update per review comments
- Use IP address instead of hostname
- Create directory needed for hdfs

elingg committed:
Add comment for configuration

elingg committed:
Update executors and conf per review

elingg committed:
Clean up / updates per review

- Return task running message after task has been launched
- Remove unnecessary code from scheduler
- Add TODO’s

elingg committed:
update log statement

elingg committed:
Merge pull request #22 from mesosphere/refactor-classes

Add multiple executors

elingg committed:
Add initial host constraints

-Simplify host constraints and add V1 of code (uses separate offers for
NN1, NN2, journal node, and data nodes).
-Remove resource code that’s currently being used for hostname
constraints

elingg committed:
Remove unused class

elingg committed:
Add constants/ Add test cases for cluster state

elingg committed:
update code to use constants

elingg committed:
Merge pull request #25 from mesosphere/refactor-classes

Add initial host constraints

elingg committed:
Add camel case to HDFS framework

CamelCase issue - should be nameNode not name node

adam-mesos committed:
Merge pull request #27 from mesosphere/refactor-classes

Add camel case to HDFS framework

elingg committed:
Fix JAVA_HOME SCRIPT to support more linux distros

Add if else clause.

If the bin/java directory exists in a specific location, we know that
it is debian/ubuntu. Otherwise, it is rehl/centos

elingg committed:
Clean up bash script

Check for JAVA_HOME in 2 places, if it’s not found then exit with an
error. This supports Debian/Ubuntu and RHEL/Centos

elingg committed:
Added error log if JAVA_HOME not found

elingg committed:
Merge pull request #28 from mesosphere/refactor-classes

Fix JAVA_HOME SCRIPT to support more linux distros

elingg committed:
Avoid race conditions when launching tasks

Order of task operations is set / Removed unnecessary sleep command

elingg committed:
fix typo

elingg committed:
add more descriptive variable name

elingg committed:
node name should be followed by executor name in the tasked

elingg committed:
Minor clean up

elingg committed:
Merge pull request #29 from mesosphere/refactor-classes

Avoid race conditions when launching tasks

elingg committed:
Simplifying scheduler

Removing unnecessary code and variables

adam-mesos committed:
Remove unused dependencies

adam-mesos committed:
Sane executor cpus; remove obsolete reconcilation property

adam-mesos committed:
Registered() should not call reregistered()

adam-mesos committed:
Update readme to point to mesosphere Travis-CI

adam-mesos committed:
Improved build-hdfs script; updated README, gitignore

adam-mesos committed:
Fix README formatting

elingg committed:
Formatting

Update formatting/ this logic will be updated in the next PR as well

elingg committed:
Merge pull request #30 from mesosphere/refactor-classes

Simplifying scheduler

elingg committed:
Use only one NameNode Executor

Remove code duplication/ Clean up logic

elingg committed:
Use constant instead of magic string

elingg committed:
fix indentation

elingg committed:
fix improper formatting

adam-mesos committed:
Run 'mvn package' in build-hdfs; update README; other review feedback

adam-mesos committed:
Add build-hdfs nocompile option; make mvn clean package the default

adam-mesos committed:
Merge pull request #34 from adam-mesos/better-build

Cleanup and build improvements

adam-mesos committed:
Fix list in README

elingg committed:
Fix security issue

elingg committed:
Merge pull request #35 from mesosphere/refactor-classes

Use only one NameNode Executor

elingg committed:
Simplify Scheduler and ClusterState Logic

Remove unnecessary/messy logic/clean up variables

elingg committed:
Add needed space

adam-mesos committed:
Fix log4j exception when starting scheduler

adam-mesos committed:
Exit build-hdfs early if compile fails

adam-mesos committed:
Copy required dependency directories, rather than deleting cruft

adam-mesos committed:
Remove unnecessary sleeps

elingg committed:
Update Scheduler Code

*Changes per code review
*More scheduler simplification
*Remove additional variable since concurrent modification exception no
longer applies

Commit:
Update String to Path so the config is read
https://hadoop.apache.org/docs/current/api/org/apache/hadoop/conf/Configuration.html

adam-mesos committed:
Merge pull request #36 from mesosphere/simplify-scheduler

Simplify Scheduler and ClusterState Logic

adam-mesos committed:
Merge pull request #42 from Banno/master

Update String to Path so the config is read

adam-mesos committed:
Merge pull request #41 from adam-mesos/less-deps

Build/dependency cleanup

Commit:
removed debugging printlns

elingg committed:
Merge pull request #45 from Banno/master

removed debugging printlns

Commit:
fixed logging errors on startup

Commit:
changed to logback

Commit:
moved both slf4j version numbers to the same var

Commit:
defined basic logback.xml

elingg committed:
Break down method into two smaller methods

elingg committed:
Merge pull request #46 from Banno/master

Redirect all logging through slflog4j to logback

elingg committed:
Pass resources to decrease dependencies in method

elingg committed:
Merge pull request #47 from mesosphere/clean-up-scheduler

Break down method into two smaller methods

elingg committed:
correct typo

elingg committed:
Merge pull request #49 from mesosphere/clean-up-scheduler

correct typo

elingg committed:
Initial symbolic link commit

elingg committed:
Code clean up

elingg committed:
Clean up

Update variable names/ add comments / multiple if statements

elingg committed:
Update comment

elingg committed:
spelling error fixed

Commit:
remove existing dist dir and tgz before build

elingg committed:
Merge pull request #52 from Banno/speed-up-build-process

Speed up compile build

lamdor committed:
first test for making sure we accept offers

Commit:
testing that it accepts and declines offers until initial set

Commit:
Cleaning up differences in state and making state easier to set up in tests

Commit:
HdfsModule for Guice isn't necessary anymore

Commit:
adding test to assure that datanodes are started

elingg committed:
Make mesosphere framework mount path configurable

elingg committed:
Minor changes

-Add logging
-Update variable name
-Rename symbolic link dir to HDFS, not hadoop

elingg committed:
Merge pull request #51 from mesosphere/distribute-hdfs-binaries

Distribute hdfs binaries

Commit:
added comment to explain why the port is a system property call

Commit:
removed mesos.dir.path from the config

elingg committed:
Merge pull request #53 from Banno/tests-and-clean

Cleanup and added some scheduler tests

elingg committed:
Exit if there is an error creating sym link

elingg committed:
Add HDFS binary to PATH env variable

This allows Mesos fetcher to work properly. It also allows us to run
hdfs commands on any slave running the framework.

Commit:
moved stagingTasks to be inside of LiveState

elingg committed:
Merge pull request #55 from mesosphere/proper-exit

Add HDFS Binary to PATH to enable use of Mesos Fetcher and to run hadoop fs commands from any slave / Exit if there is an error creating sym link

elingg committed:
Merge pull request #56 from Banno/tests-and-clean

moved stagingTasks inside of LiveState

elingg committed:
Update configuration values

*Reasonable and consistent default values in config
*Update config ordering

elingg committed:
Fix bug in properties

elingg committed:
Merge pull request #57 from mesosphere/default-config-update

Default config update

elingg committed:
Formatting

Code formatting clean up

elingg committed:
Merge pull request #58 from mesosphere/format-hdfs

Formatting

elingg committed:
Add comment per review

elingg committed:
Merge pull request #60 from mesosphere/format-hdfs

Add comment per review

elingg committed:
Add instructions for resource reservations

elingg committed:
Merge pull request #61 from mesosphere/resource-reservations

Add instructions for resource reservations

elingg committed:
Update readme per review

elingg committed:
Fix spacing

elingg committed:
Update spacing/ add punctuation

elingg committed:
remove extra period

abhay-agarwal committed:
docker container with post-build config

Change your *-site.xml files before building your docker file (for now)

elingg committed:
Capitalization

elingg committed:
Merge pull request #63 from mesosphere/update-readme

Update readme per review

brndnmtthws commented:
I think you mean to open the PR against `mesosphere/hdfs`, rather than this repo.

elingg committed:
Update config file to distribute the hdfs binary

Avoids using AWS bucket

elingg committed:
fix build

elingg committed:
Update readme to link to web UI

elingg committed:
Merge pull request #64 from mesosphere/update-config-server

Update Config Server to distribute HDFS binary

elingg committed:
Add back ticks for display of Web UI in markdown

elingg committed:
add line break

elingg committed:
remove unnecessary space

elingg committed:
Merge pull request #65 from mesosphere/update-config-server

Quick formatting fix for markdown display of the Web UI readme

abhay-agarwal committed:
Working with testing fw

- I updated the executor scripts to take  $JAVA_HOME if java isn’t set
- I added a marathon config file
- I updated the persistent state to take the current time in mills

abhay-agarwal committed:
Merge branch 'master' into dockerfile
